modules/controller/rpc-controller.js

```diff
@@ -114,7 +114,9 @@ class RpcController {
         }
     }
 
-    startListeningForRpcNetworkCalls() {
+    initializeNetworkApi() {
+        this.logger.info(`Network API module enabled, server running on port ${this.config.network.port}`);
```

Comment:
Can this be a misleading maybe, because it is not server, it is more LibP2P running on port or something else? Someone who sees this log can try to send http requests to this port

Reply:
I agree, let's have "Network API module enabled on {port}"